src/main/java/com/snowflake/kafka/connector/records/RecordService.java

```diff
@@ -275,6 +275,13 @@ public Map<String, Object> getProcessedRecordForStreamingIngest(SinkRecord recor
   private Map<String, Object> getMapFromJsonNodeForStreamingIngest(JsonNode node)
       throws JsonProcessingException {
     final Map<String, Object> streamingIngestRow = new HashMap<>();
+
+    // return empty if tombstone record
```

Comment:
You probably has tested this, but all columns will be NULL for that row?

Reply:
yup tombstone records only have a RECORD_METADATA column and the rest are all null. The key is in the RECORD_METADATA
<img width="809" alt="image" src="https://github.com/snowflakedb/snowflake-kafka-connector/assets/111399397/6a2b7c86-6cc6-4ab1-94e0-b86ace0e0fc5">